### PR TITLE
Fix entry for bitwise XOR

### PR DESCRIPTION
**Pull request by k4b7**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -99,4 +99,4 @@ Function[Symbol.operator('::')] = function (lho, rho) {
 
 ### How could this even work?
 
-When parsing a piece of javascript, when the tokenizer reached an unrecognised token between one or two operands (with, perhaps a whitelist of certain allowed operator characters, for example any Unicode character betwixt U+2000–U+218F), it could search in the operator registry (perhaps at runtime? during evaluation?) for the operator, and if not found could raise a SyntaxError.
\ No newline at end of file
+When parsing a piece of javascript, when the tokenizer reached an unrecognised token between one or two operands (with, perhaps a whitelist of certain allowed operator characters, for example any Unicode character betwixt U+2000–U+218F), it could search in the operator registry (perhaps at runtime? during evaluation?) for the operator, and if not found could raise a SyntaxError.
```
Comment (k4b7) [inline README.md:102]: I didn't change anything on this line, not sure why GitHub is showing it as changed.

Comment (cval-c4q) [reply README.md:102]: Perhaps your text editor added a previously-missing \n at the end of the file.

**Review by cval-c4q — APPROVED** [review]
Straightforward fix.

There have been quite a few variants of operator overloading in the syntax in different proposals over the years, but none have made their way into the standard, which may be for performance/practical VM-related reasons.

File: README.md (hunk 1 above)
Comment (k4b7) [reply README.md:102]: I made this change using Github's editor. :(

**Pull request closed without merging**